packages/firebase/compat/index.d.ts

```diff
@@ -6018,6 +6018,25 @@ declare namespace firebase.database {
      * ```
      */
     app: firebase.app.App;
+    /**
+     * INTERNAL methods for internal-use only (tests, etc.).
+     *
+     * @const
+     */
+    INTERNAL: {
+      /**
+       * Deletes a database
+       */
+      delete: () => Promise<void>;
+      /**
+       * Force the use of websockets instead of longPolling.
+       */
+      forceWebSockets: () => void;
+      /**
+       * Force the use of longPolling instead of websockets. This will be ignored if websocket protocol is used in databaseURL.
```

Comment:
longPolling => long polling
websockets => WebSockets
websocket protocol => the WebSocket protocol
add backticks around databaseURL

Reply:
Changed.